lib/md007.js

```diff
@@ -13,12 +13,14 @@ module.exports = {
   "function": function MD007(params, onError) {
     const indent = Number(params.config.indent || 2);
     const startIndented = !!params.config.start_indented;
+    // start_indent (number; ignored if start_indented is false, else defaults to indent)
```

Comment:
This detail (eventually) belongs in `Rules.md`. It should also outline the use of this parameter.